packages/wonder-stuff-core/src/kind-error.js

```diff
@@ -124,19 +129,21 @@ export class KindError extends Error {
 
         // Set the name so we get a nice error output, like
         // KAInternalError
-        this.name = `${prefix ?? ""}${kind}Error`;
+        this.name = `${prefix ?? ""}${kind}${name ?? "Error"}`;
```

Comment:
Added the ability to specify the name of the error, so that we can have `SentryError` at the end for Sentry error stuff.

So I'm going to add this to the `SentryError` too, so that it can be overridden.
This will change what gets recorded in Sentry, so I am pondering on whete